.travis.yml

```diff
@@ -1,9 +1,16 @@
 sudo: false
-dist: trusty
 language: php
 php:
   - 5.4
   - 5.5
   - 5.6
+  - 7.0
   - 7.1
   - 7.2
+  - 7.3
+  - hhvm
```

Comment:
usually, we drop support for hhvm. More over you have allowed failures on HHVM. So tests are running for nothing except consuming some resources, and that, that kill polar bear :D